packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.tsx

```diff
@@ -27,6 +27,10 @@ const WalletsAddMoreCardBanner: React.FC<TWalletCarouselItem> = ({
     const { addWalletAccountToLocalStorage } = useSyncLocalStorageClientAccounts();
     const { localize } = useTranslations();
     const { data: isEuRegion } = useIsEuRegion();
+    const { data: landingCompany } = useLandingCompany();
+    const { data: wallets } = useWalletAccountsList();
+    const hasAnyActiveRealWallets = wallets?.some(wallet => !wallet.is_virtual && !wallet.is_disabled);
+    const shortcode = landingCompany?.financial_company?.shortcode ?? landingCompany?.gaming_company?.shortcode;
```

Comment:
Added check for both to cater for cfd-restricted and financial-restricted countries